content/projects/data-science-specific/data-wrangling/_index.md

```diff
@@ -79,25 +80,24 @@ df.apply(get_percentage, axis = 1) #axis=1 applies the function to all columns
 
-The new data frame will look something like this and it should be named `personality_score_df`:
+The new data frame will look something like this and it should be named `personality_score_totals_df`:
 
-| I am always prepared | I am easily disturbed | I am exacting (demanding) in my work | ... | Conscientiousness | Emotional_Stability |
+| I am always prepared | I am easily disturbed | I am exacting (demanding) in my work | ... | Conscientiousness | Neuroticism |
 | -------------------- | --------------------- | ------------------------------------ | --- | ----------------- | ------------------- |
 | (3, 5)               | (4, 5)                | (3, 5)                               | ... | 10                | 5                   |
 | (3, 5)               | (4, 1)                | (3, 1)                               | ... | 6                 | 1                   |
 | (3, 5)               | (4, 3)                | (3, 3)                               | ... | 8                 | 3                   |
 
-3. Import the data in [departments.csv](departments.csv). This DataFrame should be named `department_df` Merge this DataFrame with the personality score DataFrame, keeping all applicants within the department DataFrame. Use an assert statement to check that the newly created merged DataFrame has the same amount of rows as the department DataFrame and the expected number of columns. The merged DataFrame should be named `merged_personality_department_df`
+4. Import the data in [departments.csv](departments.csv) and store it in a DataFrame. Merge this DataFrame with the personality score totals DataFrame. Again, use assertion techniques from [assertive programing tricks](projects/data-science-specific/assertive-programming-tricks-for-pandas/_index.md) to check that the newly created merged DataFrame has the expected number of rows and columns. The merged DataFrame should be named `merged_personality_department_df`
 
-4. Filter the merged DataFrame so that you get only the applicants who scored less than 30 on emotional stability, conscientiousness AND agreeableness. Print the ID numbers and departments of these applicants to the screen, and also assign these applicants the tag "high_risk" in a new column. All other applicants get the tag "low_risk". The DataFrame here should be named `risk_status_df`.
+5. Filter the merged DataFrame so that you only see the applicants who scored less than 30 on emotional stability, conscientiousness AND agreeableness. Next, assign these applicants the tag "High risk" in a new column called `Risk Status`. All other applicants get the tag "Low risk". The DataFrame here should be named `risk_status_df`.
```

Comment:
What is a tag? Is that a string?

Reply:
haha, I don't know you wrote the original instructions, lol
Kidding, but I think it's obvious from the way we talk about adding a new column called "Risk Status" and assigning the specified text string values there...